ktor-client/ktor-client-apache/jvm/test/io/ktor/client/engine/apache/FakeControl.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.engine.apache
+
+import org.apache.http.nio.*
+
+internal class FakeControl : IOControl {
```

Comment:
Please consider more descriptive name like DoNothingControl or NoOpControl

Reply:
Also, don't we already such implementation? Shouldn't we fail instead of doing nothing in the test?

Reply:
Will rename. Should we try using Mockk instead?

Reply:
I don't think so, not just for a single class.